cmd/check.go

```diff
@@ -105,3 +106,30 @@ var apacheReqCmd = &cobra.Command{
 		return nil
 	},
 }
+
+var blackfireReqCmd = &cobra.Command{
+	Use:   "blackfire",
+	Short: "Test if environment is correctly set for Blackfire",
+	Long: `Will check if your environment variables contains BLACKFIRE_SERVER_ID`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if os.Getenv("BLACKFIRE_SERVER_IDA") == "" {
```

Comment:
is it worth checking for both server env vars? I know its pretty stupid to set one and not the other, but at least if we check for both we can really tell people what their problems are. What do you think?